app/src/main/java/org/oppia/app/topic/TopicActivity.kt

```diff
@@ -18,4 +22,21 @@ class TopicActivity : InjectableAppCompatActivity(), RouteToQuestionPlayerListen
   override fun routeToQuestionPlayer(skillIdList: ArrayList<String>) {
     startActivity(QuestionPlayerActivity.createQuestionPlayerActivityIntent(this, skillIdList))
   }
+
+  override fun routeToConceptCard(skillId: String) {
+    if (getConceptCardFragment() == null) {
+      val conceptCardFragment: ConceptCardFragment = ConceptCardFragment.newInstance(skillId)
+      conceptCardFragment.showNow(supportFragmentManager, TAG_CONCEPT_CARD_DIALOG)
+    }
+  }
+
+  private fun getConceptCardFragment(): ConceptCardFragment? {
+    return supportFragmentManager.findFragmentByTag(TAG_CONCEPT_CARD_DIALOG) as ConceptCardFragment?
+  }
+
+  companion object {
+    fun getConceptCardFragmentTag(): String {
```

Comment:
Suggest just making the constant internal or public, instead. Since it's a constant, it's reasonable to expose it to other classes if needed.

Reply:
Removed this function and added:

`internal const val TAG_CONCEPT_CARD_DIALOG = "CONCEPT_CARD_DIALOG"`